src/hooks/use_chat.rs

```diff
@@ -74,7 +72,7 @@ pub fn use_chat(cx: &ScopeState) -> &UseChatState {
                     .unwrap_or(&15)
                     .clone();
 
-                process(
+                let _ = process(
```

Comment:
Instead of handling the error in `map_err` which should do that ... map the error type to something else, you can do something like `if let Err(e) = process(...).await {}` to handle the result explicitly instead of making it look like it's being ignored.